src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java

```diff
@@ -177,8 +177,8 @@ public int[] getRequiredTokens() {
             Charset.defaultCharset().name())));
 
         assertWithMessage("Invalid line content")
-                .that(check.getLine(3))
-                .isEqualTo(" * I'm a javadoc");
+                .that(check.getLine(6))
+                .isEqualTo("// violation above");
```

Comment:
sorry i ment to keep `* I'm a javadoc` and just change line number, make line number to be 10. to lee validation same as before.

Reply:
done